stable/mongodb/templates/deployment-standalone.yaml

```diff
@@ -26,9 +27,11 @@ spec:
       {{- end }}
       containers:
       - name: {{ template "mongodb.fullname" . }}
-        image: {{ template "mongodb.image" . }}
+        image: "{{ .Values.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
```

Comment:
Define a function on *_helpers.tpl* for this and use it.

Reply:
ok